### PR TITLE
chore(Screenshots for eslint error logs & hound ci comment): setting-up-linters

### PR DESCRIPTION
**Pull request by KaiserPhemi**
#### What does this PR do?
-This pull request is to merge the result of setting up linters & HoundCI chores to the output branch.

#### Description of Task to be completed.
- Setting up linters
- HoundCI integration

#### How should this be manually tested?
- View HoundCI comments on PR.
- Clone repo and run `eslint` on files.

#### Any background context you want to provide?
- Linters make sure codes are style guide (airbnb) compliant.
- HoundCI does code review for every github pull requests

#### Screenshots
-ESLint error logs
eslinterrors

-HoundCI comments
houndci comments

### CODE REVIEW

File: 06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#### Output
+- **TravisCI build link**: [Link to a successful TravisCI build](https://travis-ci.org/andela-oakinwa/inverted-index)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/README.md:2]: This isn't a successful build.

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#### Output
+- **Using**: Terminal & GitHub
+- **Deliver**: two screenshots of ESLint [Airbnb](https://github.com/airbnb/javascript)  style violations in the terminal and [HoundCI](https://houndci.com/) comment on a PR
+- **With**: *independently*
+
+#### Result
+- Screenshot of ESLint violations within the output folder.
+- Screenshot of HoundCI comments within the output folder.
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md:8]: Looks good!

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#### Output
+- **Link to repo**:[Inverted Index github repository](https://github.com/andela-oakinwa/inverted-index)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:2]: Looks good!

**Review by q-ode — COMMENTED** [review]
Kindly fix your Travis build. Everything else is good to go!

File: 06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/README.md (hunk 1 above)
Comment (KaiserPhemi) [reply 06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/README.md:2]: Error fixed, build successful.
[Link to a successful TravisCI build](https://travis-ci.org/andela-oakinwa/inverted-index)

**Pull request closed without merging**